SWLOR.Game.Server/Feature/SpawnDefinition/HutlarResourceSpawnDefinition.cs

```diff
@@ -59,5 +61,43 @@ private void Valley()
                 .AddSpawn(ObjectType.Placeable, "fiberp_bush_4")
                 .WithFrequency(10);
         }
+        private void Wastes()
+        {
+            _builder.Create("RESOURCES_HUTLAR_WASTES")
+                .AddSpawn(ObjectType.Placeable, "plagionite_vein")
+                .WithFrequency(20)
+                .AddSpawn(ObjectType.Placeable, "keromber_vein")
+                .WithFrequency(5)
+
+                .AddSpawn(ObjectType.Placeable, "ancient_tree")
+                .WithFrequency(40)
+                .AddSpawn(ObjectType.Placeable, "aracia_tree")
+                .WithFrequency(10)
+
+                .AddSpawn(ObjectType.Placeable, "patch_veggies3")
+                .WithFrequency(5)
+                .AddSpawn(ObjectType.Placeable, "patch_veggies4")
+                .WithFrequency(2)
+                .AddSpawn(ObjectType.Placeable, "fiberp_bush_3")
+                .WithFrequency(40)
+                .AddSpawn(ObjectType.Placeable, "fiberp_bush_4")
+                .WithFrequency(10);
+        }
+        private void Frzcave()
```

Comment:
Can you rename this to "FrozenCave" - it's generally best to avoid acronyms or shortened versions of words for readability purposes. Please also change the RESOURCES_FRZ_CAVE identifier below too (and update any other references in the module)

Reply:
not a problem!